src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -50,6 +50,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         $user     = $request->getAttribute("user");
         $tpl_data = array(
                      'test_name' => $this->PageName,
+                     'subtest'   => $request->getAttribute('subtest') ?? '',
```

Comment:
What is this doing? It looks like it's getting overwritten by the other line below in this class.. is "subtest" or "pageclass" the right way to get it?

Reply:
Yeah you're right, this may have been the result of two different approaches. I believe that "pageclass" is correct. I'll remove this line and test.